client-tools/Dump/DumpFeature.cpp

```diff
@@ -1167,6 +1173,16 @@ Result DumpFeature::runDump(httpclient::SimpleHttpClient& client,
     if (cid == 0 || name.empty()) {
       return ::ErrorMalformedJsonResponse;
     }
+
+    // filtered based on the user-defined list of collections to be excluded
+    if (std::find(_options.collectionsToBeIgnored.begin(),
```

Comment:
I am generally not very happy with this option, because now we have `--collection` and `--ignore-collection`.
The former is an allow list and the latter is a deny list.
What is the expected behavior if a user makes use of both options? And what is the expected behavior if the same collection name is specified in both options?
This should be at least defined.
In the easiest case, we can simply forbid using `--collection` and `--ignore-collection` at the same time.

Reply:
Understandable. I would vote for the either `--collection` or `--ignore-collection` option.